src/gui/wizard/slideshow.cpp

```diff
@@ -74,7 +74,7 @@ void SlideShow::setCurrentSlide(int index)
         _animation->setDuration(SlideDuration);
         _animation->setEasingCurve(QEasingCurve::OutCubic);
         _animation->setStartValue(static_cast<qreal>(_currentIndex));
-        connect(_animation.data(), SIGNAL(valueChanged(QVariant)), this, SLOT(update()));
+        connect(_animation.data(), &QVariantAnimation::valueChanged, this, qOverload<>(&SlideShow::update));
```

Comment:
This doesn't seem like the best way to call update, why use qOverload?

Reply:
using `qOverload` because that is easier than remembering the exact syntax via `static_cast` and using this overload of `SlideShow::update` because previous code was using it